src/App.css

```diff
@@ -47,7 +47,7 @@ form {
   transform: translate(-50%, -50%);
   border-radius: 0.6rem;
   box-shadow: 0px 3px 16px 0px rgba(0, 0, 0, 0.2);
-  padding: 0 50px 20px 50px;
+  padding: 0 3rem 1.25rem 3rem;
 }
 
 label {
```

Comment:
to avoid the complexity..why don't you have a class for labels used in the atomic components..
so you don't have to be overriding stuff